### ISSUE TITLE
focus on the next article when an article is marked as read

### ISSUE THREAD

**Issue by oupala** [FreshRSS/FreshRSS#3094]
I'd like FreshRSS to focus on the next article when an article is marked as read.

FreshRSS is currently not moving the windows focus when marking an article as read. This is quite confusing when I mark a long article as read because I am facing a white page. The next unread article is far far up at the top as the page, where I am located far far at the bottom of the page.

Have I missed a particular setting in the preferences? In the settings, I've seen in the "Display" category the settings about when to mark an article as read, and other settings about what to do when clicked on "mark all as read". But I see nothing about where to focus when clicking on "mark as read" for a specific article...

**Comment by Alkarex** [comment]
@aledeg You know shortcuts better. What do you think?

**Comment by Frenzie** [comment]
I'm not in favor of that as a default.

**Comment by aledeg** [comment]
Maybe it should be a behavior of the view and not the shortcut. This way, it will work the same way when you click on the read icon

**Comment by oupala** [comment]
I agree with @aledeg that this issue is more about how the view behave by default, than on a shortcut.

**Comment by math-GH** [comment]
Can you please describe your use case step by step?
Which view do you use?
Do you use the mobile view (small screen) or the full view ("desktop" wide screen)?
How do you navigate? (touch, mouse, keyboard)

**Comment by oupala** [comment]
I am using the *middle* view of FreshRSS, on a large screen (I think these are the default settings). I mainly navigate using the mouse. The articles from FreshRSS are displayed from the most recent to the older.

When I have read an article, I click on the red enveloppe to discard the article. The next article is now on the top of the list, however the page is still focused lower in the page, especially if the previous article was quite long. I would like the focus to be on the top of the next article to read, without being obliged to go up in order to read the article from the beginning.
